src/load.js

```diff
@@ -27,6 +14,19 @@ define(['ModernizrProto'], function(ModernizrProto) {
     };
   }
 
+  /**
+   * Previously, Modernizr.load was an alias for yepnope. Since yepnope was
+   * deprecated, we removed it as well. It is not available on the website builder,
+   * this is only included as an improved warning to those who build a custom
+   * version locally.
+   *
+   * @memberof Modernizr
+   * @name Modernizr.load
+   * @access private
+   * @function load
+   * @returns {void}
```

Comment:
void !== undefined

Reply:
same as above. can change it if you like.